l10n/de.ftl

```diff
@@ -0,0 +1,33 @@
+content-feedback_question = War diese Übersetzung hilfreich?
```

Comment:
Noting here that there seem to be string IDs here that don't match anything in en-US source.